priv/repo/migrations/20170710175356_create_email.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Askbywho.Repo.Migrations.CreateEmail do
```

Comment:
Unique email makes sense from the user perspective, I added a reminder to test how this works and affects statistics on https://trello.com/c/H3Nt9wb9/21-qa-full-site-testing

Unique email testing:
If a lisa@email.com submits Adidas, Puma and Nike today, and returns tomorrow to submit Adidas, Zara and Mango, lisa@email.com should associate Adidas, Puma, Nike, Zara, and Mango. 

In our statistics Adidas should be counted once, and Lisa@email.com should be counted as one person.